demo/README.md

```diff
@@ -0,0 +1,26 @@
+# Deploy Sindri Provers
+
+This demo builds on the mini-kube deployment of [scroll-sdk](https://scroll-sdk-init.docs.scroll.xyz/en/sdk/guides/devnet-deployment/).
+
+
+## Prerequisites
+
+1. Before you begin the process below make sure that you have already completed deployment of [scroll-sdk](https://scroll-sdk-init.docs.scroll.xyz/en/sdk/guides/devnet-deployment/) using mini-kube.
+
+2. Clone the [sindri-scroll-sdk](https://github.com/Sindri-Labs/sindri-scroll-sdk) repository.
+
+
+## Install Sindri Provers
+
+1. Create copies of the values files found in `values/`
+   - Change the `-demo.yaml` to `-local.yaml`
+
+2. For each of the 3 values files (`prover-batch-local.yaml`, `prover-chunk-local.yaml`, `prover-bundle-local.yaml`) make the following modifications:
+   - Modify `"api_key": "<your-api-key>"` by replacing `<your-api-key>` with your Sindri api key.
+
+3. Identify the desired [version](https://github.com/Sindri-Labs/sindri-scroll-sdk/pkgs/container/sindri-scroll-sdk%2Fhelm%2Fscroll-proving-sindri)
+
+4. Run deploy.sh
+   ```bash
+   SINDRI_VERSION=0.0.2 ./deploy.sh
+   ```
```

Comment:
````suggestion
Now, you can launch all Scroll zkEVM proving services with one command via 
   ```bash
   SINDRI_VERSION=0.0.2 ./deploy.sh
   ```
Running `kubectl get pods` should reveal three new services, one for each circuit type.
Assuming your chain has proving tasks, you can monitor the progress of proof generation either via the logs of any prover pod or via Sindri's web front-end after logging in:
* https://sindri.app/z/scroll-tech/batch_prover/proofs
* https://sindri.app/z/scroll-tech/bundle_prover/proofs
* https://sindri.app/z/scroll-tech/chunk_prover/proofs
````

Reply:
Shoot, one more thing. Can you add a little blurb at the end here for how to shut the provers down?